src/js/component.js

```diff
@@ -1092,6 +1092,67 @@ class Component {
   }
 
   /**
+   * Get width or height of computed style
+   * @param  {String} widthOrHeight  'width' or 'height'
+   * @return {Number|Boolean} The bolean false if nothing was set
+   * @method currentDimension
+   */
+  currentDimension(widthOrHeight) {
+    let computedWidthOrHeight;
+
+    if (widthOrHeight !== 'width' && widthOrHeight !== 'height') {
+      log.warn('currentDimension only accepts width or height value');
+      return false;
```

Comment:
It probably should return `null` or `NaN` instead of a boolean.
`NaN` would make a more consistent return signature, `null` is easier to check for.